A87-grpc-metrics-xds-outlier-detection.md

```diff
@@ -0,0 +1,64 @@
+A87: gRPC OTel Metrics for xDS Outlier Detection
```

Comment:
I don't think there's actually anything OT-specific in this proposal.  These metrics will use the non-per-call metric APIs, which can in principle be used by any gRPC stats plugin.  Currently, the only such plugin to exist is the OT plugin, but there could be others in the future, and these metrics would work fine with that plugin when that happens, since there will be nothing OT-specific in the implementation of these metrics.

Given that, I think we should remove most of the references to OT in this proposal.